components/Wave/Wave.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from './wave.module.scss'
+
+export const Wave = () => {
+  return (
+    <div className={styles.wave__content}>
```

Comment:
again dont need to keep inserting the styles

Reply:
styles is necessary for styles components

Reply:
if so... would it be better to insert the style **once** like this in every component. where you just reference the name of the component?

`<div className={styles.wave}>`